.github/workflows/my-ci-pipeline.yml

```diff
@@ -31,19 +31,10 @@ jobs:
 
     - name: Docker push
       run: docker push jormondroyd/todo-app
-
-  heroku:
-    name: Heroku deployment
+  
+  webhook:
+    name: Azure Webhook
     runs-on: ubuntu-latest
     needs: dockerhub
     steps:
-    - uses: actions/checkout@v2
-    - uses: akhileshns/heroku-deploy@v3.12.12
-      with:
-        heroku_api_key: ${{secrets.HEROKU_API_KEY}}
-        heroku_app_name: "jimbob-todo-app"
-        heroku_email: "james.ormondroyd@kpmg.co.uk"
-        usedocker: true
-    
-
- 
\ No newline at end of file
+    - run: curl -dH -X POST ${{ secrets.WEBHOOK_URL }}
```

Comment:
One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400) which can lead to confusing pipeline results